src/Constraint/ConstraintResolver.php

```diff
@@ -93,11 +93,11 @@ private function resolveConstraint(RuleList $ruleList, Rule $rule): Constraint
             case Rule::RULE_FILLED:
                 return new Assert\NotBlank(['allowNull' => $ruleList->hasRule(Rule::RULE_NULLABLE)]);
             case Rule::RULE_MIN:
```

Comment:
Nice addition, could you update the documentation with this functionality?

https://github.com/123inkt/symfony-validation-shorthand/blob/master/docs/available-shorthands.md#max
https://github.com/123inkt/symfony-validation-shorthand/blob/master/docs/available-shorthands.md#min

Add that next to integer and float also date formats are accepted, and add an extra bullet to the example.